code/src/java/pcgen/io/PCGVer2Parser.java

```diff
@@ -4260,21 +4154,17 @@ else if (IOConstants.TAG_NAME.equals(childTag))
 			}
 			else if (IOConstants.TAG_FEATLIST.equals(tag))
 			{
-				for (PCGElement child : element.getChildren())
-				{
-					final String featKey =
-							EntityEncoder.decode(child.getText());
-					final Ability anAbility =
-							Globals.getContext().getReferenceContext()
-								.silentlyGetConstructedCDOMObject(
-									Ability.class, AbilityCategory.FEAT,
-									featKey);
-
-					if (anAbility != null)
-					{
-						metaFeats.add(anAbility);
-					}
-				}
+				element.getChildren()
```

Comment:
Again, unreadable to anyone who doesn't write SCALA for a living - please leave as a FOR